src/Notification_NotificationTemplate.php

```diff
@@ -133,7 +132,7 @@ public static function showForNotification(Notification $notif, $withtemplate =
 
         if (
             $canedit
-            && !(!empty($withtemplate) && ($withtemplate == 2))
+            && !(!empty($withtemplate) && ($withtemplate === 2))
```

Comment:
Most of the time, it will be a value passed from `$_GET["withtemplate"]`, so it would be a string.

```suggestion
            && !(!empty($withtemplate) && ((int)$withtemplate === 2))
```